scripts/common-functions.sh

```diff
@@ -297,7 +299,7 @@ function bench_cmd {
   # First arg is the number of transactions with input from genesis block and
   # the second is the total number of transactions to be sent.
   echo -n " cmd --commands \"send-to-all-genesis $time $time $conc $delay ./tps-sent.csv\""
-  echo -n " --configuration-key bench "
+  echo -n " --configuration-key smallbench "
```

Comment:
'smallbench' is defined for three "richmen", exactly matching the number of core nodes in this benchmark.
the configuration 'bench' was using seven "richmen", which would allow a the relays and the wallets to create blocks!